src/libnnstreamer-edge/nnstreamer-edge-util.h

```diff
@@ -52,7 +52,7 @@ extern "C" {
 #endif
 
 #define STR_IS_VALID(s) ((s) && (s)[0] != '\0')
-#define PORT_IS_VALID(p) ((p) > 0 && (p) <= 65535)
+#define PORT_IS_VALID(p) ((p) >= 0 && (p) <= 65535)
```

Comment:
Instead of changing this macro, how about getting available port when the user try to set port number 0?
inside `nns_edge_parse_port_number()`

Reply:
Thank you.